integrations/server/test_covidcast.py

```diff
@@ -665,67 +606,48 @@ def test_unique_key_constraint(self):
+            VALUES (1, 42, 96,
+              'day', 20200414, 123, 991.5, 992.5, 993.5, 20200414, 0,
+              {Nans.NOT_MISSING}, {Nans.NOT_MISSING}, {Nans.NOT_MISSING});
+          ''')
+
+    # succeed to insert different dummy data under a different time_type
     self.cur.execute(f'''
-      INSERT INTO
-        `covidcast` (`id`, `source`, `signal`, `time_type`, `geo_type`, 
-	      `time_value`, `geo_value`, `value_updated_timestamp`, 
-        `value`, `stderr`, `sample_size`, `direction_updated_timestamp`, 
-        `direction`, `issue`, `lag`, `is_latest_issue`, `is_wip`,`missing_value`,
-        `missing_stderr`,`missing_sample_size`) 
-      VALUES
-        (0, 'src', 'sig', 'day', 'county', 20200414, '01234',
-          1, 1, 1, 1, 1, 1, 20200415, 1, 1, False,
-          {Nans.NOT_MISSING}, {Nans.NOT_MISSING}, {Nans.NOT_MISSING})
+            INSERT INTO `signal_latest` (`signal_data_id`, `signal_key_id`, `geo_key_id`,
+              `time_type`, `time_value`, `value_updated_timestamp`,
+              `value`, `stderr`, `sample_size`, `issue`, `lag`, `missing_value`, `missing_stderr`,`missing_sample_size`)
+            VALUES (2, 42, 96,
+              'score', 20200414, 123, 991.5, 992.5, 993.5, 20200415, 1,
+              {Nans.NOT_MISSING}, {Nans.NOT_MISSING}, {Nans.NOT_MISSING});
     ''')
 
   def test_nullable_columns(self):
     """Missing values should be surfaced as null."""
 
     # insert dummy data
+    self.cur.execute(f'''  INSERT INTO `signal_dim` (`signal_key_id`, `source`, `signal`) VALUES (42, 'src', 'sig');  ''')
+    self.cur.execute(f'''  INSERT INTO `geo_dim` (`geo_key_id`, `geo_type`, `geo_value`) VALUES (96, 'county', '01234');  ''')
     self.cur.execute(f'''
-      INSERT INTO
-        `covidcast` (`id`, `source`, `signal`, `time_type`, `geo_type`, 
-	      `time_value`, `geo_value`, `value_updated_timestamp`, 
-        `value`, `stderr`, `sample_size`, `direction_updated_timestamp`, 
-        `direction`, `issue`, `lag`, `is_latest_issue`, `is_wip`,`missing_value`,
-        `missing_stderr`,`missing_sample_size`) 
-      VALUES
-        (0, 'src', 'sig', 'day', 'county', 20200414, '01234',
-          123, 0.123, NULL, NULL, 456, NULL, 20200414, 0, 1, False,
-          {Nans.NOT_MISSING}, {Nans.OTHER}, {Nans.OTHER})
+      INSERT INTO `signal_latest` (`signal_data_id`, `signal_key_id`, `geo_key_id`,
+        `time_type`, `time_value`, `value_updated_timestamp`,
+        `value`, `stderr`, `sample_size`, `issue`, `lag`, `missing_value`, `missing_stderr`,`missing_sample_size`)
+      VALUES (0, 42, 96,
+        'day', 20200414, 123, 0.123, NULL, NULL, 20200414, 0,
+          {Nans.NOT_MISSING}, {Nans.OTHER}, {Nans.OTHER});
     ''')
     self.cnx.commit()
 
+    # TODO: should this show that even if numeric values go into the fields, that if they are marked w/ a MISSING reason, they show up as NULL anyway?
+
```

Comment:
fwiw there's some checking in `csv_importer.py` that should prevent cases like this. now if something like that is _manually loaded_ bypassing CSVs, then it's a different matter.